### ISSUE TITLE
Work s3 bucket assumes us-east-1 region

### ISSUE THREAD

**Issue by andrewmcgilvray** [Teevity/ice#253]
Now that ice uses a per bucket region specifier for the billing bucket "ice.billing_s3bucketregion" if your work bucket is not in us-east-1 ice cannot access it. A new config is needed to specify the work bucket region : "ice.work_s3bucketregion"

**Comment by NitriKx** [comment]
You are right. Until we had a "ice.work_s3bucketregion", the working bucket must be in the same region as the billing bucket (so that they can share the same existing region config).

**Comment by multani** [comment]
Probably a duplicate of #215
